app/views/hosts/_list.html.erb

```diff
@@ -3,7 +3,7 @@
 <table class="<%= table_css_classes 'table-fixed' %>">
   <thead>
     <tr>
-      <th class="ca" width="40px"><%= check_box_tag "check_all", "", false, { :onclick => "toggleCheck()", :'check-title' => _("Select all items in this page"), :'uncheck-title'=> _("items selected. Uncheck to Clear") } %></th>
+      <th class="ca" width="40px"><%= check_box_tag "check_all", "", false, { :onchange => "toggleCheck()", :'check-title' => _("Select all items in this page"), :'uncheck-title'=> _("items selected. Uncheck to Clear") } %></th>
```

Comment:
While you're here, a couple of minor points:
1. check-title should be "Select all items on this page"
2. uncheck-title is never shown (perhaps we should just show "Toggle all items on this page"?) and also should be rephrased.
3. When clicking the tooltip isn't removed until another click.

I won't block on this, but perhaps it's time to make the checkboxes a react component? looks like a lot of the js code is quite old and over-complicated.